PDF_UA/2/8.2 Logical structure/8.2.5 Additional requirements for specific structure types/8.2.5.29 Formula/verapdf-profile-8-2-5-29-t02.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<profile xmlns="http://www.verapdf.org/ValidationProfile" flavour="PDFUA_2">
+    <details creator="veraPDF Consortium" created="2023-03-13T10:53:06.894+03:00">
+        <name>ISO 14289-2:202x - 8.2 Logical structure - 8.2.5.29 Formula - Non-math formula</name>
+        <description>Non-mathematical scientific formula enclosed within Formula structure elements shall have either 
+            an Alt entry or an ActualText entry in its structure element dictionary</description>
+    </details>
+    <hash></hash>
+    <rules>
+        <rule object="SEFormula">
+            <id specification="ISO_14289_2" clause="8.2.5.29" testNumber="2"/>
+            <description>Non-mathematical scientific formula enclosed within Formula structure elements shall have either
+                an Alt entry or an ActualText entry in its structure element dictionary</description>
+            <test>Alt != null || ActualText != null || isMathFormula == true</test>
+            <error>
+                <message>Non-mathematical scientific formula enclosed within Formula structure elements does not contain 
```

Comment:
A non-mathematical scientific formula enclosed within Formula structure elements contains neither Alt nor ActualText entry in its structure element dictionary